src/vs/editor/browser/standalone/standaloneLanguages.ts

```diff
@@ -85,28 +85,28 @@ export function setMonarchTokensProvider(languageId: string, languageDef: IMonar
  * Register a reference provider (used by e.g. reference search).
  */
 export function registerReferenceProvider(languageId: string, provider: modes.ReferenceProvider): IDisposable {
-	return modes.ReferenceProviderRegistry.register(languageId, provider);
+	return modes.ReferenceProviderRegistry.register(languageId, null, provider);
```

Comment:
Is it possible to make the workspace parameter to register the last parameter and make it optional, to reduce the size of the diff? Then it would be undefined if not explicitly specified, which seems equivalent to what you are manually adding here.